### DISCUSSION TITLE
Something went wrong. Please try again a bit later.

### DISCUSSION THREAD

**Discussion by drkamine** [umputun/remark42#1119] [Q&A] [answered]
![errors](https://user-images.githubusercontent.com/48792324/131631826-4147ec97-a98f-4874-80ba-55cb5bc9f833.jpg)
![errors](https://user-images.githubusercontent.com/48792324/131631856-9dd5e3f7-953b-4bfa-8a95-ee82607b8b72.jpg)
Hello ,  
  i just install a docker version latest . when i do a comment i have this error 
Something went wrong. Please try again a bit later.

**Comment by umputun** [comment, marked as answer]
this is not enough info to help you. I would suspect misconfigured auth, but can't tell for sure without additional details, like your configuration settings.

_this is not a bug report, but rather some configuration issue; moved to discussions_

**Reply by umputun** [reply]
also, see the log on the backend side, i.e. `docker logs remark42`. It should have details about the reason for 5xx errors
